test/e2e/pages-dir/next-dynamic/with-static-import/index.test.ts

```diff
@@ -0,0 +1,51 @@
+import { nextTestSetup } from 'e2e-utils'
+
+describe('css-module-with-next-dynamic-and-static-import', () => {
```

Comment:
we should have some test for `next start` to test out the build manifest

Reply:
Added here: https://github.com/vercel/next.js/pull/68396/commits/a5bd4e108ea9043a4ea4ae56192b71295f47c4d1